packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -165,6 +166,47 @@ export const useArbTokenBridge = (
     updateEthBalances()
   }
 
+  const depositEthFromContract = async (weiValue: BigNumber) => {
+    const etherVal = utils.formatUnits(weiValue, 18)
+
+    const inboxAddress = bridge.l1Bridge.network.ethBridge?.inbox;
```

Comment:
can't we instead do `bridge.l1Bridge.getInbox()` and get an instantiated inbox contract wrapper?